csrc/ir/interface_nodes.h

```diff
@@ -550,33 +546,30 @@ class NVF_API TensorView : public Val {
   }
 
  private:
-  int64_t normalizeAxisPos(int64_t pos) const {
-    if (pos < 0) {
-      pos += (int64_t)nDims();
-    }
-    return pos;
+  int64_t wrapDim(int64_t dim) const {
+    return nvfuser::wrapDim(dim, nDims());
   }
 
   //! A helper function to maintain the consistency of schedules of
   //! multiple outputs wheen doing rfactor on multi-output reduction ops.
-  TensorView* multiOutputRfactorHelper(
+  TensorView* multiOutputrFactorHelper(
```

Comment:
so I think `RFactor` should be the consistent one instead.

Reply:
Changed